### DISCUSSION TITLE
drag-drop-ID required

### DISCUSSION THREAD

**Discussion by eFastRun** [community/community#45793] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Body

I used react-beautiful-dnd for drag and drop page.
Here I set the ID and key correctly and these are unique.
But I am getting the ID required error like the below.
![image](https://user-images.githubusercontent.com/123271974/215484262-9c8a759e-d9d4-4d2d-bd6b-19046e8a0ea7.png)

**Comment by SuperHeroDev** [comment, marked as answer]
I faced this error in the past and solved by updated ID.
This is because of wrong ID. I suggest you to change the ID.
